util/chplenv/third_party_utils.py

```diff
@@ -15,12 +15,11 @@
 #
 @memoize
 def default_uniq_cfg_path():
-    arch_val = chpl_arch.get('target', map_to_compiler=True,
-                             get_lcd=using_chapel_module()).arch
-    return '{0}-{1}-{2}-{3}'.format(chpl_platform.get('target'),
-                                    chpl_compiler.get('target'),
-                                    arch_val,
-                                    chpl_lib_pic.get())
+    pic = chpl_lib_pic.get()
+    result = chpl_bin_subdir.get('target')
+    if pic != 'none':
```

Comment:
Why change this to not include pic if it's none?

Reply:
It makes sense to me to omit it if it's none but we don't have to keep that change.